packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -484,23 +557,30 @@ class IconButton extends StatelessWidget {
         adjustedStyle = style!.merge(adjustedStyle);
       }
 
+      Widget adjustedIcon = icon;
```

Comment:
We usually refer to these computed values as `effectiveIcon` or something like that.